modules/representation_management/app/services/representation_management/address_validation_service.rb

```diff
@@ -62,19 +64,23 @@ def validate_address(address_hash)
 
     # Builds a VAProfile ValidationAddress object from an address hash
     #
+    # Uses Veteran::AddressPreprocessor to normalize PO Boxes and suite/room information
+    #
     # @param address_hash [Hash] Address data with string keys
     # @return [VAProfile::Models::ValidationAddress]
     def build_validation_address(address_hash)
+      cleaned = Veteran::AddressPreprocessor.clean(address_hash)
```

Comment:
Rails engines/modules should be self-contained and not have any dependencies in the host app or in other engines.

This file should be copied over to the `representation_management` module - `modules/veteran/app/services/veteran/address_preprocessor.rb` and referenced.